assets/app/features/upgrade/controllers/upgrade-landing.controller.spec.js

```diff
@@ -91,6 +110,33 @@ describe('Upgrade Landing Controller', function() {
                     assert.isFalse(value.status);
                 });
             });
+
+            it('should contain checks for addons', function () {
+                expect(controller.prechecks.checks).toEqual(
+                    jasmine.objectContaining({
+                        'clusters_healthy': jasmine.objectContaining(
+                            { status: jasmine.any(Boolean), label: jasmine.any(String) }
+                        ),
+                        'ceph_healthy': jasmine.objectContaining(
+                            { status: jasmine.any(Boolean), label: jasmine.any(String) }
```

Comment:
I think it would be a nice to have so we don't mess around language keys without noticing (making unit tests fail).
But I don't consider it's a blocker for these changes to be merged, so up to you